tests/compat.sh

```diff
@@ -888,7 +888,7 @@ if type lsof >/dev/null 2>/dev/null; then
     }
 else
     wait_server_start() {
```

Comment:
I had a one-line patch to make diagnosis easier, it could be merged together with this PR: #1260